relay-general/src/store/normalize.rs

```diff
@@ -104,6 +104,15 @@ impl<'a> NormalizeProcessor<'a> {
         })
     }
 
+    /// Ensure measurements interface is only present for transaction events
+    fn normalize_measurements(&self, event: &mut Event) {
+        if event.ty.value() != Some(&EventType::Transaction) {
+            // Only transaction events may have a measurements interface
+            event.measurements = Annotated::empty();
+            return;
```

Comment:
This `return` statement is redundant and can be removed.

Unrelated: This function looks good and is integrated at the right place 👍

Reply:
Fixed in https://github.com/getsentry/relay/pull/724/commits/d4b9040ae92aa98264faf297e96636e945733fff